apps/etterlatte-pdltjenester/src/main/kotlin/pdl/PdlOboKlient.kt

```diff
@@ -6,16 +6,20 @@ import io.ktor.client.HttpClient
 import io.ktor.client.call.body
 import io.ktor.client.request.accept
 import io.ktor.client.request.bearerAuth
+import io.ktor.client.request.header
 import io.ktor.client.request.post
 import io.ktor.client.request.setBody
 import io.ktor.http.ContentType
 import io.ktor.http.contentType
 import no.nav.etterlatte.libs.common.RetryResult
 import no.nav.etterlatte.libs.common.behandling.SakType
+import no.nav.etterlatte.libs.common.person.Folkeregisteridentifikator
+import no.nav.etterlatte.libs.common.person.PersonRolle
 import no.nav.etterlatte.libs.common.retry
 import no.nav.etterlatte.libs.ktor.behandlingsnummer
 import no.nav.etterlatte.libs.ktor.ktor.ktorobo.AzureAdClient
 import no.nav.etterlatte.libs.ktor.token.BrukerTokenInfo
+import no.nav.etterlatte.utils.toPdlVariables
 import org.slf4j.LoggerFactory
 
 class PdlOboKlient(private val httpClient: HttpClient, private val config: Config) {
```

Comment:
Lage et interface for Pdl klientene, bruk interfacet i parameterne, slipper mye unødvendig dobbel kode i mapperne

Reply:
Blir kanskje mer søl en det er verdt